app/src/main/java/com/example/coinage/fragments/GoogleSheetsFragment.java

```diff
@@ -217,9 +215,9 @@ private void createSpreadsheet(HttpRequestInitializer requestInitializer) throws
     }
 
     // export transactions to google sheets asynchronously
-    private class ExportTransactions extends AsyncTask<Void, Void, Void> {
+    private class ExportTransactions extends AsyncTask<Void, Void, Boolean> {
         @Override
-        protected Void doInBackground(Void... voids) {
+        protected Boolean doInBackground(Void... voids) {
```

Comment:
nit - add a comment about what the return value means, it is not implicit with something like this